vignettes/admiral.Rmd

```diff
@@ -77,7 +117,7 @@ dataset_vignette(
 )
 ```
 
-**Note:** We only display variables that were derived.  A user running this code will have additional adsl variables displayed.  
+**Note:** We only display variables that were derived.  A user running this code will have additional adsl variables displayed. You can use the **Choose Columns to Display** button to add more variables into the Table.  
 
 # Derivations
```

Comment:
I couldn't comment on the exact line since it was not updated, but in line 130-134 maybe we can also mention the difference between `derive_var_xxx` vs. `derive_vars_xxx` and similarly param vs. params. What do you think?

Reply:
so the one derive_params_exposure is being deprecated, so no need for an explanation for derive_params